contracts/VotingEscrow.vy

```diff
@@ -490,9 +506,47 @@ def withdraw():
 
     assert ERC20(self.token).transfer(msg.sender, value)
 
+    if not _unlocked:
+      IVeRBNRewards(self.reward_pool).updateReward(msg.sender) # Reward pool snapshot
+
     log Withdraw(msg.sender, value, block.timestamp)
     log Supply(supply_before, supply_before - value)
 
+@external
+@nonreentrant('lock')
+def force_withdraw():
```

Comment:
We have to check that `self.reward_pool` is not `address(0)` so we do not inadvertently burn tokens